department-of-veterans-affairs/vets-website
# Update content with ConfirmationView component

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Update the confirmation page on the Pension Benefits form (21P-527EZ)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/98092

## Associated Pull Request(s)
n/a

## References
- [ConfirmationView Component](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/platform/forms-system/src/js/components/ConfirmationView)
- [Figma Design](https://www.figma.com/design/9JKK5Eo43uJWEr66JPiebc/WIP---21P-527EZ---Pension-Benefits?node-id=9392-54150&t=tzz3rkWvFXU5kWKn-4)

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `pension_confirmation_update ` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_confirmation_update
4. Go to `http://localhost:3001/pension/apply-for-veteran-pension-form-21p-527ez/` in your browser

## How to verify
1. Submit the application with mock data via the SIP menu
2. Verify the layout, style and content changes match the [design](https://www.figma.com/design/9JKK5Eo43uJWEr66JPiebc/WIP---21P-527EZ---Pension-Benefits?node-id=9392-54150&t=tzz3rkWvFXU5kWKn-4)

## What areas of the site does it impact?
Pension Benefits Application

## Screenshots
### Desktop
#### Confirmation Page
| Current      | Updated       | Updated (Unauthenticated) |
| ----------- | ----------- | ----------- |
|![ConfirmationPage-Old(new)](https://github.com/user-attachments/assets/b47bb4ba-3601-4313-83c2-9ec7c48c2399)|![ConfirmationPage-New(new)](https://github.com/user-attachments/assets/dc35ce49-ba78-4a69-b580-64aa9046e6cf)|![ConfirmationPage-New-Unauth(new)](https://github.com/user-attachments/assets/a413cb93-6d6c-4f4a-87ae-3c304d4370a4)


### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
